src/models/propulsion/FGBldc.h

```diff
@@ -0,0 +1,100 @@
+/*%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+ Header:       FGBldc.h
```

Comment:
```suggestion
 Module:       FGBrushlessDCMotor.h
```
Please use a more explicit name for the file such as `FGBrushlessDCMotor.h`